README.md

```diff
@@ -1 +1,460 @@
-# golangcodestyle
\ No newline at end of file
+# Backend (Golang) Coding Guidelines
```

Comment:
Also consider: 
https://hackernoon.com/you-dont-understand-the-single-responsibility-principle-abfdd005b137
https://medium.com/code-smells/layered-architecture-f11bc04c5d6c

(I am lazy, those are one of the first links I found about single responsibility principle and code layering. Both topics should be understood before designing code.

Reply:
added: https://github.com/apaliavy/golangcodestyle/blob/1eda2a9506765ca11274f4bf973baf16a84174af/README.md#before-you-start